mindmap/public/js/translations/es_419/text.js

```diff
@@ -0,0 +1,203 @@
+
+            (function(global){
+                var MindMapI18N = {
+                  init: function() {
+                    
+
+'use strict';
+{
+  const globals = this;
+  const django = globals.django || (globals.django = {});
+
+  
+  django.pluralidx = function(n) {
+    const v = (n != 1);
+    if (typeof v === 'boolean') {
+      return v ? 1 : 0;
+    } else {
+      return v;
+    }
+  };
+  
+
+  /* gettext library */
+
+  django.catalog = django.catalog || {};
+  
+  const newcatalog = {
```

Comment:
I recommend to use one file for this translations and each respective translation inside the same object:

something like this

```
 const newcatalog =  {
   "ES_419":{
      "Actions":"Acciones",
      "Back":"Volver"
   },
   "es_ES":{
      "Actions":"Acciones",
      "Back":"Volver"
   }
}
```